pages/issue.jsx

```diff
@@ -10,6 +10,6 @@ export default function (props) {
   return <div>
             <Helmet><title>{issueName}</title></Helmet>
             <IssueSelector />
-            <ProjectLoader issue={encodeURIComponent(issueName)} showCounter={true} />
+            <ProjectLoader issue={issueName} showCounter={true} />
```

Comment:
oh man, is there a way for us to add a test to the code that will catch things like this in the future?